corehq/apps/zapier/api/v0_5.py

```diff
@@ -0,0 +1,34 @@
+import json
+
+from corehq.apps.api.resources.v0_4 import XFormInstanceResource
+from corehq.apps.api.serializers import XFormInstanceSerializer
+from corehq.apps.export.system_properties import MAIN_FORM_TABLE_PROPERTIES
+
+
+class ZapierXFormInstanceSerializer(XFormInstanceSerializer):
+
+    def to_json(self, data, options=None):
+        form_json = json.loads(super(ZapierXFormInstanceSerializer, self).to_json(data, options))
+        forms = form_json['objects']
+
+        for form in forms:
```

Comment:
is there not a way in tastypie to override the object level serializer as opposed to working with the group as a second pass? that would feel cleaner to me. if that is not possible, I think it would at least be good to pull the inside of this loop out into a function so it's clear why it's here and what it does